### PR TITLE
Add Cronosphere as sponsor

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

File: content/2020-vancouver/index.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ Get an impression of PromCon EU 2019:
 
 Registration for PromCon NA 2020 will open soon.
 
+## Our sponsors
+
+A big thank you to all of our sponsors who are helping make PromCon NA 2020 happen!
```
Comment (roidelapluie) [inline content/2020-vancouver/index.md]: Should we add a link here to the page with the prospectus?

File: content/2020-vancouver/index.md [hunk 2]
```diff
@@ -16,6 +16,15 @@ Get an impression of PromCon EU 2019:
...
+
+<h3>Diamond</h3>
+<div class="sponsor-logos">
+  <a href="https://chronosphere.io/"><img src="/assets/Chronosphere_vertical_logo.svg" class="logo"/></a>
```
Comment (roidelapluie) [inline content/2020-vancouver/index.md]:
```suggestion
  <a href="https://chronosphere.io/"><img alt="Chronosphere" src="/assets/Chronosphere_vertical_logo.svg" class="logo"/></a>
```

File: content/2020-vancouver/index.md (hunk 1 above)
Comment (RichiH) [reply content/2020-vancouver/index.md]: No, that's linked from the sponsor page.

**Comment by roidelapluie** [comment]
I dont see a link to sponsors on https://deploy-preview-139--promcon.netlify.com/

**Comment by roidelapluie** [comment]
Sorry, your branch is not rebased on top of master. LGTM!

**Pull request merged**